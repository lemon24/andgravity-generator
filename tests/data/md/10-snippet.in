
.. snippet:: one
    :key: value
    *text*
    more text

.. snippet:: two
