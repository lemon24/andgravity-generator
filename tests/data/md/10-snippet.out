snippet: one
options: {'key': 'value'}
text: '*text*\nmore text\n\n'
snippet: two
options: {}
text: ''
